docs/usage.md

```diff
@@ -838,52 +838,29 @@ For GATK.GRCh38 the links for each reference file and the corresponding processe
 
-## How to customise snpeff and vep annotation
+## How to customise snpEff and VEP annotation
 
-### Using the nf-core containers with pre-downloaded cache
-
-For common genomes, it is already configured within the [igenomes.config](https://github.com/nf-core/sarek/blob/master/conf/igenomes.config) file, so nothing to be done there.
-
-Note: These containers are only created for some species and some cache/tools versions combinations (cf DockerHub tags for these containers [`nfcore/snpeff`](https://hub.docker.com/r/nfcore/snpeff/tags) and [`nfcore/vep`](https://hub.docker.com/r/nfcore/vep/tags).
-
-These containers can be quite huge especially for human, it is recommended to use annotation cache on a path if possible
-
-### Create containers with pre-downloaded cache
-
-For each tool, an helper script `build.sh` can be found at the root of the tool folder in the nf-core module repo ([snpeff](https://github.com/nf-core/modules/tree/master/modules/nf-core/snpeff) and [ensemblvep](https://github.com/nf-core/modules/tree/master/modules/nf-core/ensemblvep)), and can be adapted for your usage.
-
-### Use Sarek to download cache and annotate in one go
-
-Use the params `--download_cache`, and specify with `--tools` for which annotation tool you need to download the cache (`snpeff` and or `vep`)
-
-Sarek will automatically download the cache, use the biocontainers container for said tools, and use it to annotate any vcfs produced.
-
-### Only download cache
-
-Using the params `--build_only_index` allow for only downloading the cache for the specified tools.
+SNPeff and VEP require a large resource of files known as a cache. These are folders composed of multiple gigabytes of files which need to be available for the software to properly function. To use these, supply the parameters `--vep_cache` and/or `--snpeff_cache` to the relevant folders.
 
-### Location for the cache
+### Specify the cache location
 
-Cache can be downloaded in the specified `--outdir_cache` location.
-Else, it will be downloaded in `cache/` in the specified `--outdir` location.
-
-To download cache on a cloud infrastructure, an absolute path is needed.
+Params `--snpeff_cache` and `--vep_cache` are to used to specify the locations to the root of the annotation cache folder. The cache will be located within a subfolder with the path `${vep_species}/${vep_genome}_${vep_cache_version}` for VEP and `${snpeff_species}.${snpeff_version}` for snpEff. If this directory is missing, Sarek will raise an error.
 
-Params `--snpeff_cache` and `--vep_cache` are to used to specify the locations to the root of the annotation cache folder.
+For example this is a typical folder structure for GRCh38 and WBCel235, with SNPeff version 105 and VEP version 106:
```

Comment:
where does the 106 come from?

Reply:
Good point, these are not the tools' version, but the tools' cache version